examples/plugins/traffic-control/report.go

```diff
@@ -263,18 +263,29 @@ func getControls() []extControl {
 				Rank:  22,
 			},
 			handler: func(pid int) error {
-				return DoTrafficControl(pid, "1ms")
+				return DoTrafficControl(pid, "1ms", "")
+			},
+		},
+		{
+			control: control{
+				ID:    "pkt-drop-low",
+				Human: "Packet drop: low",
+				Icon:  "fa-cut",
+				Rank:  23,
+			},
+			handler: func(pid int) error {
+				return DoTrafficControl(pid, "", "10%")
```

Comment:
Won't setting the packet drop to low clear the latency setting? and vice-versa? They seem like they should be independent.